bower.json

```diff
@@ -12,7 +12,7 @@
   "dependencies": {
     "n-ui-foundations": "^2.5.0",
     "o-teaser": "^2.0.0",
```

Comment:
If o-teaser is not also bumped to v3.x might this cause version conflicts?

https://github.com/Financial-Times/o-teaser/blob/v2-backporting/bower.json#L19